mappings/net/minecraft/entity/mob/warden/WardenEntity.mapping

```diff
@@ -0,0 +1,70 @@
+CLASS net/minecraft/unmapped/C_fynaplbn net/minecraft/entity/mob/warden/WardenEntity
+	FIELD f_aklkpyqr lastHeartAnimationTicks I
+	FIELD f_azgepkis sniffingAnimation Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_bglhpwqx DIGGING_PARTICLE_OFFSET F
+	FIELD f_bokpqluc DARKNESS_DISPLAY_LIMIT I
+	FIELD f_catvnevx VIRBRATION_COOLDOWN_TICKS I
+	FIELD f_ceomecbq ANGER_BOOST_ON_HURT I
+	FIELD f_ctsqwubn emergingAnimation Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_ejgyiohn ATTACK_KNOCKBACK F
+	FIELD f_fmvpjdup PROJECTILE_ANGER I
+	FIELD f_gmstpsno ANGER_TICK_DELAY I
+	FIELD f_gqdmtuqw MAX_HEALTH I
+	FIELD f_jaxcsvmw gameEventListener Lnet/minecraft/unmapped/C_wpaexsvv;
+	FIELD f_jctlypxu roaringAnimation Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_jgqxnrro DEFAULT_ANGER_LEVEL I
+	FIELD f_jhswzlgv tendrilAnimationTicks I
+	FIELD f_knxwlbyj DARKNESS_INTERVAL I
+	FIELD f_kvtjaaga DIGGLE_PARTICLE_DURATION F
```

Comment:
DIGGLE?

Reply:
My typing skill is something to behold. Should be DIGGING and am fixing now